packages/sdks/src/blocks/columns/columns.lite.tsx

```diff
@@ -24,6 +28,8 @@ export interface ColumnProps {
   stackColumnsAt?: StackColumnsAt;
   // TODO: Implement this when support for dynamic CSS lands
   reverseColumnsWhenStacked?: boolean;
+
+  customBreakpoints?: Breakpoints;
```

Comment:
Ok, I think I see how this works!

It's a good solution, but I'm wondering if we can avoid the need for an extra prop altogether. We should be able to read this value from the Builder context! We already consumer the context in other blocks like Symbol, Form.

```
export default function Columns(props: ColumnProps) {
  const builderContext = useContext(BuilderContext);

// inside getWidthForBreakpointSize
  const breakpointSizes = getSizesForBreakpoints(
        builderContext.content?.meta?.breakpoints || {}
      );
```

This way, you don't have to pass `customBreakpoints` around in 3 different components. What do you think?

Reply:
We could have used it, but when I was working on it, I didn't see context being used in columns.lite. so came up with this approach. I needed it to be cleaner and didn't want to stuff things in context again.
But this can be done, provided it works without surprises, but in another PR? later pls??